README.md

```diff
@@ -1 +1,105 @@
-# Countries
\ No newline at end of file
+### Space Travelers's Hub
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+## About the Project
+
+> This project is built with React.js, it is a page of fetching API for Countries in the World. On this website you can look, search and categorize countries by region.
+
+## Built With
+
+- React.js
+- Gitflow
+- Redux
+- API
+- React Router Dom
+- Redux Toolkit
+
+### Key Features
+
+- SPA
+- Search for countries
+- Look for countries throught the regions
+- Webpage fetchin api countries
+- UI
+- Mobile first design
+- Desktop media query design
+
+## Live Demo
+
+Live demo:[Click here](https://country-enis.onrender.com/)
+
+## Video Link
+
+- [Click here](https://www.loom.com/share/e2f4096ae8e14dc4877accbe516ec684)
+
+## Getting Started
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-prefer-folder
+  git clone https://github.com/enis-memic/Countries
+```
+
+### Run this command to run server
+
+> npm start
+
+### Run this command build server
+
+> npm run build
+
+### Run this command to run tests
+
+> npm run test
+
+## 👥 Authors <a name="authors"></a>
```

Comment:
- [ ] Kindly add the command to install the project packages for it to run successfully after cloning it. `npm install` 👍🏽

Reply:
Thank you!